src/main/java/apoc/mongodb/MongoDB.java

```diff
@@ -70,9 +70,17 @@ private Coll getColl(@Name("host") String hostOrKey, @Name("db") String db, @Nam
     }
 
     @Procedure
-    @Description("apoc.mongodb.find(host-or-port,db-or-null,collection-or-null,query-or-null,projection-or-null,sort-or-null,[compatibleValues=true|false]) yield value - perform a find,project,sort operation on mongodb collection")
-    public Stream<MapResult> find(@Name("host") String hostOrKey, @Name("db") String db, @Name("collection") String collection, @Name("query") Map<String, Object> query, @Name("project") Map<String, Object> project, @Name("sort") Map<String, Object> sort, @Name(value = "compatibleValues", defaultValue = "false") boolean compatibleValues) {
-        return getMongoColl(hostOrKey, db, collection, compatibleValues).find(query, project, sort).map(MapResult::new);
+    @Description("apoc.mongodb.find(host-or-port,db-or-null,collection-or-null,query-or-null,projection-or-null,sort-or-null,pagination,[compatibleValues=true|false]) yield value - perform a find,project,sort operation on mongodb collection")
+    public Stream<MapResult> find(@Name("host") String hostOrKey, 
+                                  @Name("db") String db, 
+                                  @Name("collection") String collection, 
+                                  @Name("query") Map<String, Object> query, 
+                                  @Name("project") Map<String, Object> project,
+                                  @Name("sort") Map<String, Object> sort, 
+                                  @Name(value = "compatibleValues", defaultValue = "false") boolean compatibleValues,
+                                  @Name(value = "skip", defaultValue = "0") Long skip,
+                                  @Name(value = "limit", defaultValue = "0") Long limit) {
```

Comment:
default value should be -1

Reply:
A limit() value of 0 (i.e. .limit(0)) is equivalent to setting no limit.

https://docs.mongodb.com/manual/reference/method/cursor.limit/#cursor.limit

Reply:
Ah ok. :)